api-docs.md

```diff
@@ -561,6 +563,102 @@ There are, however, a few catches to this:
+3. Providing access to the [results](#get-ting-results) via the `jobs/TICKET/results` resource.
+
+#### GET-ting summary of all jobs
+A user may get the status of all jobs by sending a `GET` to `jobs` endpoint.
+
+```
+https://HOST:PORT/v1/jobs
+```
+If no jobs are available in the system, an empty collection is returned.
+
+The `jobs` endpoint supports filtering on job fields (i.e. `userId`, `status`), using the same syntax as the
+[data endpoint filters](#filtering). For example:
+
+userId-eq[greg, joan], status-eq[success]
+
+resolves into the following boolean operation:
+
+(userId = greg OR userId = joan) AND status = success
+
+which will return only those Jobs created by `greg` and `joan` that have completed successfully.
+
+#### GET-ting Job Status
+When the user sends a `GET` request to `jobs/TICKET`, Fili will look up the specified ticket and return the job's
+meta-data as follows:
+
+###### Job meta-data
+```json
+{
+    "query": "https://HOST:PORT/v1/data/QUERY",
+    "results": "https://HOST:PORT/v1/jobs/TICKET/results",
+    "syncResults": "https://HOST:PORT/v1/jobs/TICKET/results?asyncAfter=never",
+    "self": "https://HOST:PORT/v1/jobs/TICKET",
+    "status": ONE OF ["pending", "success", "error"],
+    "jobTicket": "TICKET",
+    "dateCreated": "ISO 8601 DATETIME",
+    "dateUpdated": "ISO 8601 DATETIME",
+    "userId": "Foo"
+}
+```
+* `query` is the original query
+* `results` provides a link to the data, whether it is fully synchronous or switches from
+   synchronous to asynchronous after a timeout depends on the default setting of `asyncAfter`.
+* `syncResults` provides a synchronous link to the data (note the `asyncAfter=never` parameter)
+* `self` provides a link that returns the most up-to-date version of this job
+* `status` indicates the status of the results pointed to by this ticket
+    - `pending` - The job is being worked on
+    - `success` - The job has been completed successfully
+    - `error` - The job failed with an error
+    - `canceled` - The job was canceled by the user (coming soon)
+* `jobTicket` is a unique identifier for the job
+* `dateCreated` is the date on which the job was created
+* `dateUpdated` when the job's status was last updated
+* `userId` is an identifier for the user who submitted this job
+
+If the ticket is not available in the system, we get a 404 error with the message `No job found with job ticket TICKET`
+
+#### GET-ting Results
+The user may access the results of a query by sending a `GET` request to `jobs/TICKET/results`. This
+resource takes the following parameters:
+
+ 1. **`format`** - Allows the user to specify a response format, i.e. csv, or JSON. This behaves
+just like the [`format`](#response-format) parameter on queries sent to the `data` endpoint.
+
+2. **`page`, `perPage`** - The [pagination](#pagination--limit) parameters. Their behavior is the same as when sending
+a query to the `data` endpoint, and allow the user to get pages of the results.
```

Comment:
I wouldn't worry about it too much. We'll have it shortly, and this is a patch version, not a stable version, so it's OK for the docs to be a bit ahead of the implementation.